.husky/pre-commit

```diff
@@ -2,5 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn type-check
-yarn prettier --check .
-yarn eslint .
\ No newline at end of file
+yarn lint-staged
```

Comment:
Missing a newline at the end of file. There seems to be a quite a bit of these in this repo, so I'd suggest creating a file called `.editorconfig` to the root of the repo with just the content
```
root = true

[*]
insert_final_newline=true
```
So you can automatically have \n at the end of each file.

Reply:
Will open a new PR for this. I'm assuming this is mostly a problem with files that aren't being formatted by prettier (dot files and json files).